getting-started/setup-guide.md

```diff
@@ -140,7 +152,9 @@ class Forest::OrdersController < ForestLiana::SmartActionsController
 end
 ```
 
-Note that Forest Admin takes care of the authentication thanks to the `ForestLiana::SmartActionsController` parent class controller.
+{% hint style="warning" %}
+You must make sure that all your Smart Actions controllers extend from the `ForestLiana::SmartActionsController`. This is mandatory to ensure that all features built on top of Smart Actions work as expected (authentication, approval workflows,...)
```

Comment:
```suggestion
You must make sure that all your Smart Actions controllers extend from the `ForestLiana::SmartActionsController`. This is mandatory to ensure that all features built on top of Smart Actions work as expected (authentication, permissions, approval workflows,...)
```

Reply:
Fun fact, I added it in the before your comment :)

Reply:
I just saw it 🎯